src/features/layers_in_area/atoms/areaLayersDetailsResource/areaLayersDetailsResourceAtom.ts

```diff
@@ -8,6 +8,7 @@ export const areaLayersDetailsResourceAtom = createAsyncAtom(
   areaLayersDetailsParamsAtom,
   async (params, abortController) => {
     if (params === null) return null;
```

Comment:
not your change, but it looks like this line is no more needed as params can't be null, isn't it?

Reply:
Cool!  Extra check removed